src/arcade/potts/env/location/PottsLocation.java

```diff
@@ -627,141 +631,36 @@ Location separateVoxels(ArrayList<Voxel> voxelsA, ArrayList<Voxel> voxelsB,
         calculateCenter();
         return makeLocation(voxelsB);
     }
-    
+
     /**
-     * Splits the voxels in the location along a given direction.
-     *
-     * @param direction  the direction of the slice
-     * @param voxels  the list of voxels
-     * @param voxelsA  the container list for the first half of the split
-     * @param voxelsB  the container list for the second half of the split
-     * @param center  the center voxel
-     * @param random  the seeded random number generator
-     */
-    static void splitVoxels(Direction direction, ArrayList<Voxel> voxels,
+     * Divides the voxels in the location into two lists along a given plane.
```

Comment:
```suggestion
     * Splits the voxels in the location into two lists along a given plane.
```

For consistency with the name of the method